scripts/travis-ci.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+if [ "$TRAVIS_OS_NAME" == "linux" ]; then
+    set -e
+    docker build --tag bitbox-wallet-dev -f Dockerfile .
+    docker build --tag bitbox-wallet-ci -f Dockerfile.travis .
+    docker run -i bitbox-wallet-ci bash \
+        -c "make -C \$GOPATH/src/github.com/digitalbitbox/bitbox-wallet-app ci" &
+    while [ -e /proc/$! ]; do echo -n "."  && sleep 60; done
+    docker run --privileged -i bitbox-wallet-ci bash \
+        -c "make -C \$GOPATH/src/github.com/digitalbitbox/bitbox-wallet-app qt-linux"
+    set +e
+fi
+
+if [ "$TRAVIS_OS_NAME" == "osx" ]; then
+    brew install go
```

Comment:
replace all of this with `make osx-init`?

Reply:
I think the first step would break as brew is already installed on the system.

Reply:
Could make it not fail somehow, or just remove it (make a small note in the README that brew is required).